src/commands/chooseRevisionMode.ts

```diff
@@ -8,45 +8,51 @@ import { ProgressLocation, window } from "vscode";
 import { IActionContext, IAzureQuickPickItem } from "vscode-azureextensionui";
 import { RevisionConstants } from "../constants";
 import { ext } from "../extensionVariables";
+import { ContainerAppTreeItem } from "../tree/ContainerAppTreeItem";
 import { RevisionsTreeItem } from "../tree/RevisionsTreeItem";
 import { createContainerAppsAPIClient } from "../utils/azureClients";
 import { localize } from "../utils/localize";
 import { nonNullValue } from "../utils/nonNull";
 
-export async function chooseRevisionMode(context: IActionContext, node?: RevisionsTreeItem): Promise<void> {
+export async function chooseRevisionMode(context: IActionContext, node?: ContainerAppTreeItem): Promise<void> {
```

Comment:
sigh.... yeeeeeaah.  But I changed it because in this scope:

![image](https://user-images.githubusercontent.com/5290572/160726526-d513e047-d353-4907-9140-7cd4c5491e6b.png)

 Typescript forgets that I already determined that this is a `ContainerAppItem` and throws typing errors.